testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/IdentityProviderTest.java

```diff
@@ -1071,8 +1074,22 @@ public void testSamlExportSignatureOn() throws URISyntaxException, IOException,
         response.close();
 
         Document document = DocumentUtil.getDocument(body);
+        System.out.println(DocumentUtil.getNodeAsString(document));
```

Comment:
Please remove this line. I think it's just a leftover.

Reply:
Fixed in the updated PR.